packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -944,10 +979,15 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
 
     return itemToString(value)
   }
+
+  private isValueEmpty = (value: ShorthandValue | ShorthandValue[]) => {
```

Comment:
haaa! we needed the function now :)
does it need to be part of the class though?

Reply:
Going to leave it as class method as it is very coupled with the component.